packages/lab/src/stepped-tracker/Step.css

```diff
@@ -0,0 +1,59 @@
+.saltStep {
+  /* Copy of size calculations of <Icon /> */
+  --icon-base-size: var(--salt-size-icon, 12px);
```

Comment:
Not sure why icon CSS needs re-defining. Try delete and see what happens.

If there has to be custom variable to handle certain things, use step icon specific variable, not all icons, otherwise custom icon placed in the stepper will be impacted as well

Reply:
Me and @origami-z discussed this yesterday, for everybody else reading this PR, these variables are defined on a component level (`<Icon />`), but I need in a higher component for grid and placement of connector purposes. The variables are necessary for the component function, but the naming might not be consistent, so I switched to --step-icon-[...rest] nameing